### ISSUE TITLE
Is there a Sigmoid Cross Entropy Loss Layer(like the caffe one) in mxnet?

### ISSUE THREAD

**Issue by wangg12** [dmlc/mxnet#961]

**Comment by chonglinsun** [comment]
You may use
```python
mx.symbol.LogisticRegressionOutput(data = ..., label = ..., name = ...)
```

**Comment by wangg12** [comment]
There is little tutorial about this. I'm not sure the caffe one can be found here. In addition, the name of `LogisticRegressionOutput` is hard to understand, I don't know what function it actually does.

**Comment by zhangchen-qinyinghua** [comment]
I agree with that.
I think it will be better if MXNET separated the output operator apart from loss operator.

There's no categorical entropy loss for MXNET.
Considering it's just a kind of sum of log operator,  it should be easy to implement it.
However, the softmax output operater mix loss and output together, and becomes an objective layer.
That means the backward of softmax-outpus operator is not the general meaning of softmax.

The reason to mix softmax and softmax-loss is it's more efficient, I guess?

**Comment by piiswrong** [comment]
@zhangchen-qinyinghua we have a softmax activation without loss in cudnn-activation

**Comment by zhangchen-qinyinghua** [comment]
@piiswrong 
Do we have that?
I only find the following:
https://github.com/dmlc/mxnet/blob/master/src/operator/cudnn_activation-inl.h#L18-L35

**Comment by tqchen** [comment]
related to #472 this is not yet implemented

**Comment by piiswrong** [comment]
Its in my PR. Forgot I havent merged yet

**Comment by wangg12** [comment]
I ended up using
 caffe due to time limit and confusion about the layer defining in mxnet.
